Packs/DeprecatedContent/Scripts/script-VectraTriage.yml

```diff
@@ -21,7 +21,7 @@ type: python
 subtype: python2
 tags:
 - Vectra
-comment: Deprecated. The rules branch can be used to retrieve a listing of configured Triage rules
+comment: The rules branch can be used to retrieve a listing of configured Triage rules
```

Comment:
why did you remove the deprecated note here?

Reply:
By mistake. Fixed.